packages/admin-ui/ui/public/locales/th/translation.json

```diff
@@ -0,0 +1,2011 @@
+  "edit-new-total": "ผลรวมใหม่",
+  "edit-difference-due": "ความแตกต่างเนื่องจาก",
+  "edit-back": "กลับ",
+  "edit-save-and-go-back": "บันทึกและย้อนกลับ",
+  "edit-order-edit-set-as-requested": "สั่งแก้ไขชุดตามที่ร้องขอ",
+  "edit-failed-to-request-confirmation": "ไม่สามารถขอคำยืนยันได้",
+  "edit-added-successfully": "เพิ่มเรียบร้อยแล้ว",
+  "edit-error-occurred": "เกิดข้อผิดพลาด",
+  "edit-add-product-variants": "เพิ่มตัวเลือกสินค้า",
+  "edit-edit-order": "แก้ไขคำสั่งซื้อ",
+  "edit-items": "รายการ",
+  "edit-add-items": "เพิ่มรายการ",
+  "edit-filter-items": "กรองรายการ...",
+  "edit-note": "บันทึก",
+  "edit-add-a-note": "เพิ่มบันทึก...",
+  "variants-table-location": " ใน {{location}}",
+  "edit-product": "ผลิตภัณฑ์",
+  "edit-in-stock": "มีสินค้า",
+  "edit-price": "ราคา",
+  "edit-products": "สินค้า",
+  "edit-search-product-variants": "ค้นหาผลิตภัณฑ์ย่อย...",
+  "orders-success": "สำเร็จ",
+  "orders-successfully-initiated-export": "เริ่มการส่งออกเรียบร้อยแล้ว",
+  "orders-error": "ข้อผิดพลาด",
+  "orders-export-orders": "ส่งออกคำสั่งซื้อ",
+  "components-billing-address": "ที่อยู่เรียกเก็บเงิน",
+  "components-use-same-as-shipping": "ใช้แบบเดียวกับการขนส่ง",
+  "components-e-g-gift-wrapping": "เช่น การห่อของขวัญ",
+  "components-title": "ชื่อ",
+  "components-price": "ราคา",
+  "components-quantity": "ปริมาณ",
+  "components-back": "กลับ",
+  "components-add": "เพิ่ม",
+  "components-items-for-the-order": "รายการสั่งซื้อ",
+  "components-details": "รายละเอียด",
+  "components-price-excl-taxes": "ราคา (ไม่รวมภาษี)",
+  "components-add-custom": "เพิ่มแบบกำหนดเอง",
+  "components-add-existing": "เพิ่มที่มีอยู่",
+  "components-add-products": "เพิ่มสินค้า",
+  "components-add-custom-item": "เพิ่มรายการที่กำหนดเอง",
+  "components-choose-region": "เลือกภูมิภาค",
+  "components-region": "ภูมิภาค",
+  "select-shipping-to-name": "(ชื่อ}})",
+  "components-attention": "ข้อควรระวัง!!",
+  "components-no-options-for-orders-without-shipping": "คุณไม่มีตัวเลือกใดๆ สำหรับคำสั่งซื้อที่ไม่มีการจัดส่ง โปรดเพิ่มตัวเลือกหนึ่ง (เช่น \\การดำเนินการในร้านค้า\\) โดยไม่ได้เลือก \\แสดงบนเว็บไซต์\\ ในการตั้งค่าภูมิภาคแล้วดำเนินการต่อ",
+  "components-choose-a-shipping-method": "เลือกวิธีการจัดส่ง",
+  "components-set-custom-price": "กำหนดราคาเอง",
+  "components-custom-price": "ราคาที่กำหนดเอง",
+  "components-customer-and-shipping-details": "รายละเอียดลูกค้าและการจัดส่ง",
+  "components-find-existing-customer": "ค้นหาลูกค้าปัจจุบัน",
+  "components-email": "อีเมล",
+  "components-choose-existing-addresses": "เลือกที่อยู่ที่มีอยู่",
+  "components-create-new": "สร้างใหม่",
+  "components-the-discount-is-not-applicable-to-the-selected-region": "ส่วนลดไม่สามารถใช้ได้กับภูมิภาคที่เลือก",
+  "components-the-discount-code-is-invalid": "รหัสส่วนลดไม่ถูกต้อง",
+  "components-add-discount": "เพิ่มส่วนลด",
+  "components-summer-10": "ซัมเมอร์ 10",
+  "components-discount": "การลดราคา",
+  "select-shipping-code": "(รหัส: {{รหัส}})",
```

Comment:
The same comment goes for `{{รหัส}}` too.

Reply:
Fixed by https://github.com/medusajs/medusa/pull/5654/commits/89161bb7c15fd57c652b4030df95c7474ec74974